testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/remote/common/EJBManagementUtil.java

```diff
@@ -267,7 +267,7 @@ public static void createRemoteOutboundConnection(final ModelControllerClient mo
             addRemoteOutboundConnection.get("outbound-socket-binding-ref").set(outboundSocketRef);
             addRemoteOutboundConnection.get(SECURITY_REALM).set("PasswordRealm");
             addRemoteOutboundConnection.get("username").set("user1");
-            addRemoteOutboundConnection.get("protocol").set("http-remoting");
+            addRemoteOutboundConnection.get("protocol").set("remote+http");
```

Comment:
Is that related to domain configuration?

Reply:
http-remoting is deprecated in favor of remote+http